ts/features/idpay/code/screens/WizardScreen.tsx

```diff
@@ -0,0 +1,109 @@
+import {
```

Comment:
Can we move this file to `ts/components/screens`?
I think it's a generic component.

Reply:
Addressed into d4095f06a313075110e6af132741a40a80275339